experimental/dds/tree2/src/test/feature-libraries/default-field-kinds/defaultChangeFamily.spec.ts

```diff
@@ -365,43 +359,6 @@ describe("DefaultEditBuilder", () => {
 			expectForest(forest, expected);
 		});
 
-		describe("encodes insert ops using schema based encoding", () => {
```

Comment:
This file is testing the in memory representation, which no longer needs access to the schema, not does compressed encoding, so this test can be removed.